src/glue/jobs/s3_to_json.py

```diff
@@ -130,10 +130,16 @@ def get_metadata(basename: str) -> dict:
     if metadata["type"] == "HealthKitV2Samples":
         metadata["subtype"] = basename_components[1]
     if (
-            metadata["type"] == "HealthKitV2Samples"
-            and basename_components[-2] == "Deleted"
-        ):
-        metadata["type"] = "HealthKitV2Samples_Deleted"
+        metadata["type"]
+        in [
+            "HealthKitV2Samples",
+            "HealthKitV2Heartbeat",
+            "HealthKitV2Electrocardiogram",
+            "HealthKitV2Workouts",
+        ]
+        and basename_components[-2] == "Deleted"
+    ):
+        metadata["type"] = "{}_Deleted".format(metadata["type"])
```

Comment:
Have you seen f-strings? They're super slick: https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

Reply:
Nit: I definitely prefer to use f-string when possible, but personally, the double/single quote is annoying

```
f"{metadata['type']}_Deleted"
```

we can...


```
meta_type = metadata['type']
f"{meta_type}_Deleted"
```

I am also ok with `.format` in this specific scenario.

Reply:
I have seen it, but I just thought in this scenario that it would look a bit cleaner to have .format.